AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/CascadeDeleteOperation.swift

```diff
@@ -346,6 +368,7 @@ public class CascadeDeleteOperation<M: Model>: AsynchronousOperation {
                                associatedModels: [(ModelName, Model)],
                                syncEngine: RemoteSyncEngineBehavior,
                                completion: @escaping DataStoreCallback<Void>) {
+        self.log.debug("[CascadeDelete.4] Begin syncing  \(models.count) \(modelName) model for deletion")
```

Comment:
this looks out of place, i think when `associatedModels.isEmpty` (two lines below) then we call a private `syncDeletions` method that has no `asssociatedModels` parameter. This log line should go in there